### ISSUE TITLE
Static x values if y values at index are null?

### ISSUE THREAD

**Issue by msuprovici** [kevinzhow/PNChart#46]
For PNBarChart and PNLineChart, the x axis values expand dynamically based on the array length.  Is there a way to keep the x values static even though some of the y at index are null?  

For example: Say I wanted the x values to be the days of the week (Mon - Sun) and they are static.  But today is Tuesday and I only have y values for Mon  & Tue.  Because the for the rest of the days (Wed - Sun) the y values are null, the chart will crash.  I can create the logic to dynamically add 0 for each day left in the array, but this approach is not very scalable.  PNLineChart will also draw a line through the 0 values, which is also not ideal.

Do you have any advice here?  BTW, terrific job with this library.

**Comment by aminas** [comment]
Hi msuprovici,

I'm already having the same problem and I went for the same approach and this is not fun to watch where you want to see the trend of data.

Any idea? What did you do at last?

**Comment by msuprovici** [comment]
aminas - I insert 0s to a mutable array for PNBarChart.  I don't use PNLineChart because it draws a line through the 0 values at the end, creating a bad UI.

**Comment by aminas** [comment]
Hmm that's a good idea because the bars wont show up. But what if we have more than one series of data. In case of line chart you get more lines but for bar chart wouldn't be messy?

**Comment by msuprovici** [comment]
Yup, very messy.  This approach is not scalable.  I re-architected my data around the limitations of the library:  I show only one x-value for each y-value.

**Comment by aminas** [comment]
Not good. I'm sure this is doable in PNChart. There's a lableWidth = chartWidth / [data count]; somewhere which makes the fixed intervals. This must be change. I wish I could have enough time in this project. Tnx btw :]

**Comment by aaronisme** [comment]
Same Problem, Do you have some idea how to fix this?
